tests/README.md

```diff
@@ -84,6 +84,16 @@ user=lizmap
 password=lizmap1234!
 ```
 
+Admin User Account Setup
+========================
+
+Default admin credentials are `admin`/`admin`, to modify it, modify these variables in `docker-compose.yml` : 
```

Comment:
the modification should not occurs into docker-compose.yml. The goal is to avoid to modify files. 

You should indicate to create environment variables before launching run-docker